popSimManu.tex

```diff
@@ -24,12 +22,12 @@
 
 % method names
 
-\newcommand{\stdpopsim}{\texttt{stdpopsim} }
-\newcommand{\dadi}{$\partial a \partial i$ }
-\newcommand{\MSMC}{\texttt{MSMC} }
-\newcommand{\smcpp}{\texttt{smcpp} }
-\newcommand{\stairwayplot}{\texttt{stairwayplot} }
-\newcommand{\fastsimcoal}{\texttt{fastsimcoal} }
+\newcommand{\stdpopsim}{\texttt{stdpopsim}\xspace}
+\newcommand{\dadi}{$\partial a \partial i$\xspace}
+\newcommand{\MSMC}{\texttt{MSMC}\xspace}
+\newcommand{\smcpp}{\texttt{smcpp}\xspace}
+\newcommand{\stairwayplot}{\texttt{stairwayplot}\xspace}
+\newcommand{\fastsimcoal}{\texttt{fastsimcoal}\xspace}
```

Comment:
this makes it ok to do, for instance, `\stdpopsim, a simulation library`.

Reply:
ah nice

Reply:
Very nice, been wondering about how to do that!

Reply:
The xspace author recommends against using `\xspace`.
https://tex.stackexchange.com/questions/86565/drawbacks-of-xspace

Reply:
ok, so I guess we should leave out the space and remember to put `{}` after these.